app/src/main/java/com/github/se/polyfit/ui/screen/recipeRec/RecipeRecommendationScreen.kt

```diff
@@ -43,14 +50,23 @@ fun recipeDisplay(recipesRec: RecipeRecommendationViewModel) {
 
   LazyColumn(
       horizontalAlignment = Alignment.CenterHorizontally,
-      modifier = Modifier.testTag("RecipeList")) {
+      modifier = Modifier.testTag("RecipeList").padding(paddingValues)) {
         item {
           Text(
               text = ContextCompat.getString(context, R.string.recommendedRecipe),
               style = MaterialTheme.typography.displaySmall,
               fontWeight = FontWeight.Bold,
           )
         }
-        recipes.value.forEach { item { RecipeCard(recipe = it) } }
+        recipes.value.forEach {
```

Comment:
Maybe you want to use you might want to use a nested `LazyColumn` or `LazyRow` aswell

Reply:
But it is already in a ` LazyColumn`. The `item` describes rows in the column.